src/Gauge/ThermalAssistantRenderer.cpp

```diff
@@ -17,6 +17,13 @@
 
 #include <algorithm> // for std::clamp()
 
+static void
+DrawCircleLabel(Canvas &canvas, PixelPoint p,
+                tstring_view text) noexcept;
+
+static void
+DrawCircleLabelVSpeed(Canvas &canvas, PixelPoint p, double value) noexcept;
```

Comment:
Undeclared? What does that mean?

Reply:
perhaps my terminology is wrong. Initially the 'PaintRadarPlane' radar function did not have access to these functions. My initial approach was to move the function but this caused diff problems. Therefore I have declared the function at the top. please can you suggest a better method?

Reply:
You can move functions if they need to be moved. Just make it a separate commit (and let the commit message explain why this move is needed), or else you confuse reviewers.
Adding unnecessary forward declarations just clutters the code.

Reply:
roger. thanks for your help. sorted